src/test/kotlin/com/atlassian/performance/tools/hardware/HwrDatasetCatalogue.kt

```diff
@@ -139,16 +133,23 @@ class HwrDatasetCatalogue {
         label = "1M issues JSW 8 MySQL",
         databaseDownload = Duration.ofMinutes(20),
         jiraHomeDownload = Duration.ofMinutes(20)
-    ).overrideDatabase { original ->
-        overrideLicense(original)
-    }.let { dataset ->
+    ).let { dataset ->
         AdminDataset(
-            dataset = dataset,
+            dataset = fix(dataset),
             adminLogin = "admin",
             adminPassword = "admin"
         )
     }
 
+    private fun fix(
+        dataset: Dataset
+    ): Dataset = dataset
+        .overrideDatabase {
+            overrideLicense(it)
+        }.overrideJiraHome {
+            AppNukingJiraHome(it.jiraHomeSource)
+        }
+
```

Comment:
do you need to override the jira with correct version of JSW?

Reply:
No need
A `ProductDistribution` installs JSW